pkg/multiTenancyPlugins/naming/nameScoping.go

```diff
@@ -129,3 +132,147 @@ func (nameScoping *DefaultNameScopingImpl) Handle(command utils.CommandEnum, clu
 	}
 	return nil
 }
+
+func CheckContainerReferences(cluster cluster.Cluster, tenantId string, containerConfig *dockerclient.ContainerConfig) (error) {
+	log.Debugf("CheckContainerReferences containerConfig: %+v",containerConfig)
+	// create arrays of container references to pass to getIDsFromContainerReferences
+	// create volumesFrom array	
+	volumesFrom := containerConfig.HostConfig.VolumesFrom
+	
+	// create affinity array
+	affinityRef := make([]string,0)
+	env := containerConfig.Env
+	var affinityIndex int
+	affinityLabelFirst := false
+	for i, e := range env {
+		if strings.HasPrefix(e,"affinity:") {
+		  if strings.HasPrefix(e,"affinity:image==") {
+			break  // ignore affinity for images 
+		  } else if strings.HasPrefix(e,"affinity:container==") {
+			containerRefIndex := strings.Index(e,"affinity:container==")+len("affinity:container==")
```

Comment:
IMHO this is to fragile a minor change in text, encoding or indentation will break this code. Use regexps or other way maybe?